openmetadata-spec/src/main/resources/json/schema/entity/data/table.json

```diff
@@ -14,9 +14,21 @@
       "description": "Name of a table. Expected to be unique within a database.",
       "type": "string",
       "minLength": 1,
-      "maxLength": 128,
+      "maxLength": 512,
```

Comment:
This is too big no?, max is 256 by snowflake right?

Reply:
did a check and 516 is super long and it would require us to allocate more space on the `name` field in the db. 256 looks like a good compromise and is aligned with the rest of the entities